package.json

```diff
@@ -16,35 +16,27 @@
   },
   "author": "shilman",
   "license": "MIT",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/ts/index.d.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
   "bin": {
-    "test-storybook": "./bin/test-storybook.js"
+    "test-storybook": "./dist/test-storybook"
```

Comment:
Is it OK to have no file-extension here?

Reply:
Yes. Node automatically uses test-storybook.js for cjs projects and test-storybook.mjs for esm projects.